src/components/Editor/CodeEditor/index.tsx

```diff
@@ -131,6 +159,9 @@ export default function CodeEditor({ forwardedRef }: CodeEditorProps) {
               updateCursor(actor, editor.posFromIndex(0));
             }
           }
+        } else if (event.type === 'snapshot') {
+          setOnChangesHandler();
+          editor.setValue(doc.getRoot().content.toString());
```

Comment:
🤔 The way of registering `Text.onChange` handler looks dirty.

Reply:
@Eithea Thanks for posting the issue for this.
https://github.com/yorkie-team/yorkie-js-sdk/issues/357